deploy.sh

```diff
@@ -0,0 +1,22 @@
+# prepare beckend part 
+cd /Users/Mikhail_Proshin/work/nestjs-rest-api;
+npm run build;
+# prepeare frontend part
+cd /Users/Mikhail_Proshin/work/shop-react-redux-cloudfront/
+rm -rf ./dist;
+npm run build;
+# clear data
+ssh 13.53.61.166;
+rm /var/www/html/*;
+rm -rf /home/ubuntu/nestjs-rest-api;
+exit;
+# transfer data
+scp /Users/Mikhail_Proshin/work/nestjs-rest-api ubuntu@13.53.61.166:/home/ubuntu;
+scp /Users/Mikhail_Proshin/work/shop-react-redux-cloudfront/dist ubuntu@13.53.61.166:/var/www/html
+# run new version
+ssh 13.53.61.166;
+cd /home/ubuntu/nestjs-rest-api;
+npm run buld;
```

Comment:
did you try to execute this file

Reply:
no, actually I made some changes in code on instance without changing source. That's why I did not execute it in order not to loose it. So now after you've seen that it worked. I can check the script